test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -208,6 +208,18 @@ function (Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthForm, Be
     return setup(settings);
   }
 
+  function setupPIV (pivAuthentication, useDefaultText) {
+    var buttonText = (useDefaultText) ? '' : 'piv test text';
+    var settings = {
+      piv: {
```

Comment:
do we have a case where users can pass piv config without the url. shouldnt setup(false) have settings without piv object ?

Reply:
Same comment as above